modules/dop/initialize.go

```diff
@@ -315,6 +306,14 @@ func initEndpoints(db *dao.DBClient) (*endpoints.Endpoints, error) {
 		branchrule.WithDBClient(db),
 		branchrule.WithBundle(bdl.Bdl),
 	)
+	fileTree := filetree.New(filetree.WithBundle(bdl.Bdl), filetree.WithBranchRule(branchRule))
```

Comment:
variable name should not as same as pkg name

Reply:
uodate fileTree to gittarFileTreeSvc